irs-testing/src/main/java/org/eclipse/tractusx/irs/testing/wiremock/DiscoveryServiceWiremockSupport.java

```diff
@@ -79,20 +91,35 @@ public static MappingBuilder postDiscoveryFinder200() {
                 responseWithStatus(STATUS_CODE_OK).withBody(discoveryFinderResponse(EDC_DISCOVERY_URL)));
     }
 
-    public static String discoveryFinderResponse(final String discoveryFinderUrl) {
+    public static MappingBuilder postDiscoveryFinder200(final String... edcUrls) {
```

Comment:
maybe reuse this method in postDiscoveryFinder200() with EDC_DISCOVERY_URL. As far as I can see `postDiscoveryFinder200(final String... edcUrls)` is not used anywhere